modules/porous_flow/paper.bib

```diff
@@ -22,6 +22,7 @@ @TechReport{pflotran
 Author = "Peter C. Lichtner and Glenn E. Hammond and Chuan Lu and Satish Karra and Gautam Bisht and Benjamin Andre and Richard T. Mills and Jitendra Kumar and Jennifer M. Frederick",
 Title = "{PFLOTRAN} {U}ser {M}anual",
 Note = "http://documentation.pflotran.org",
+url = "http://documentation.pflotran.org",
```

Comment:
```suggestion
url = "https://documentation.pflotran.org",
```
Might as well so modern browsers don't complain. (Soon, they should all attempt to promote.)

Reply:
Fixed, thanks